scripts/log-template.yaml

```diff
@@ -1,6 +1,6 @@
 rotation:
-    logLimit: 2097152 # 2MB
-    keepFiles: 7
+    logLimit: 104857600 # 100 MB
```

Comment:
Well, I wanted to use `memory-units` here. But we decided that `log-warper` shouldn't depend on `serokell-core`. Probably this change will took place when `memory-units` move into separate lib.